nixos/modules/services/desktops/pipewire/pipewire.nix

```
@@ -332,7 +332,7 @@ in {
       {
         # JACK intentionally not checked, as PW-on-JACK setups are a thing that some people may want
         assertion = (cfg.alsa.enable || cfg.pulse.enable) -> cfg.audio.enable;
-        message = "Using PipeWire's ALSA/PulseAudio compatibility layers requires running PipeWire as the sound server. Set `services.pipewire.audio.enable` to true.";
+        message = "Using PipeWire's ALSA/PulseAudio compatibility layers requires running PipeWire as the sound server. Set `services.pipewire.audio.enable` to true, or set both `services.pipewire.pulse.enable` and `services.pipewire.alsa.enable` to false if you wish to use pipewire only for compositor.";
```

```suggestion
        message = "Using PipeWire's ALSA/PulseAudio compatibility layers requires running PipeWire as the sound server. Set `services.pipewire.audio.enable` to true, or set both `services.pipewire.pulse.enable` and `services.pipewire.alsa.enable` to false if you wish to use pipewire only for video.";
```

Would `video` be a bit easier to understand here compared to `compositor`?

---

> Would `video` be a bit easier to understand here compared to `compositor`?

Hmm I agree that the term compositor is a bit too contextual, but `video` is too inaccurate IMO and a bit confusing - people might think it may only be related to video displaying programs like `vlc`, and not the basic functionality of the desktop environment etc. I'd prefer to be accurate and less immediately clear for inexperienced users, but I'm open to hear other suggestions.